loop/management/commands/maharashtra_add_agg_mandi.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
+from loop.models import LoopUser, Mandi, LoopUserAssignedMandi
+
+import pandas as pd
+
+class Command(BaseCommand):
+
+	def handle(self, *args, **options):
+		df = pd.DataFrame()
+
+		df = pd.read_excel('Maharashtra Admin Data Sheet.xlsx', sheetname='Aggregator - Markets')
```

Comment:
Shall we continue code execution if no sheet available named 'Aggregator - Markets'.